code/datums/components/edit_complainer.dm

```diff
@@ -0,0 +1,23 @@
+// This is just a bit of fun while making an example for global signal
+/datum/component/edit_complainer
+	var/list/say_lines
+
+/datum/component/edit_complainer/Initialize(list/text)
+	if(!ismovableatom(parent))
+		return COMPONENT_INCOMPATIBLE
+
+	var/static/list/default_lines = list(
+		"CentCom's profligacy frays another thread.",
+		"Another tug at the weave.",
+		"Who knows when the stresses will finally shatter the form?",
+		"Even now a light shines through the cracks.",
+		"CentCom once more twists knowledge beyond its authority.",
+		"There is an uncertain air in the mansus.",
+		)
+	say_lines = text || default_lines
```

Comment:
say_lines = LAZYLEN(text) ? text : default_lines

Reply:
if someone provides an empty list that's their own damn fault